jina/peapods/runtimes/base.py

```diff
@@ -11,21 +11,19 @@ class BaseRuntime:
      In the sequel, we call the main process/thread as ``M``, the process/thread blocked :class:`Runtime` as ``S``.
 
      In Jina, a :class:`BasePea` object is used to manage a :class:`Runtime` object's lifecycle. A :class:`BasePea`
-     is a subclass of :class:`multiprocessing.Process` or :class:`threading.Thread`, it starts from ``M`` and once the
+     acts as a :class:`multiprocessing.Process` or :class:`threading.Thread`, it starts from ``M`` and once the
      ``S`` is spawned, it calls :class:`Runtime` methods in the following order:
 
-        0. :meth:`__init__` in ``M``
+        0. :meth:`__init__`
 
-        1. :meth:`setup` in ``S``
-
-        2. :meth:`run_forever` in ``S``. Note that this will block ``S``, step 3 won't be
+        2. :meth:`run_forever`. Note that this will block ``S``, step 3 won't be
```

Comment:
Change the number here
```suggestion
        1. :meth:`run_forever`. Note that this will block ``S``, step 3 won't be
```